src/routes/books/books.router.js

```diff
@@ -0,0 +1,48 @@
+const express = require("express");
+const { check } = require("express-validator");
+
+const {
+  listBooks,
+  postBook,
+  updateBook,
+  deleteBook
+} = require("./books.controller");
+const { validateBody } = require("../../middleware/validate-body");
+
+const router = express.Router();
+
+const booksRequestBodyCheck = [
```

Comment:
Could you think of additional things we can check here to make the validation more robust? For example, should we allow `rating_counts` to be a string?